cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
         return data
```

Comment:
In the `validate` method, you are calling the `validate_ticket` method from the `Ticket` model. This method seems to be validating the ticket based on the row, seat, and cinema hall. However, it's not clear what kind of validation is being performed. It would be helpful to add a comment explaining what kind of validation is being performed and why it's necessary.